packages/@react-aria/toast/src/useToast.ts

```diff
@@ -25,8 +25,10 @@ export interface AriaToastProps<T> extends AriaLabelingProps {
 }
 
 export interface ToastAria {
-  /** Props for the toast container element. */
+  /** Props for the toast container, non-modal dialog element. */
   toastProps: DOMAttributes,
+  /** Props for the toast content alert message. */
+  contentProps: DOMAttributes,
```

Comment:
Do we still need both contentProps and titleProps as separate elements or are they the same thing now?

Reply:
yeah, it still provides a useful distinction for the aria-labelledby. You can put things like the icon on the outside of the title so it isn't read off all the time

Reply:
Is the new `contentProps` required to be used? The documentation isn't really clear about this. I currently don't have a wrapper element around my toast title and description as they can be arranged differently in layout with CSS Grid.

Reply:
It probably depends a bit on how complex your layout is. It can be helpful to target the live announcer to a more succinct announcement. So I'd say good practice, but possibly not strictly necessary. You'll have to test it out with a screen reader to see if not including it is sufficient.

Reply:
@snowystinger I see. Thanks for the response!

Reply:
@levrik and @snowystinger The new `contentProps` includes the `role="alert"` attribute, which prompts live region announcement when the Toast appears, so I would say these props are necessary.